packages/api/lib/schemas.js

```diff
@@ -625,6 +625,11 @@ module.exports.provider = {
       title: 'Concurrent Connection Limit',
       type: 'integer',
     },
+    maxDownloadTime: {
+      title: 'Maximum download time in seconds for all granule files on a sync granule task',
+      type: 'integer',
+      description: 'The timeout is used together with globalConnectionLimit to limit concurrent downloads',
+    },
```

Comment:
Nice! Should we update https://github.com/nasa/cumulus/blob/master/docs/configuration/data-management-types.md?plain=1#L63 as well?

Reply:
Added